README.md

```diff
@@ -57,6 +57,38 @@ It doesn't check browser/runtime-specific APIs (see [eslint-plugin-compat](https
 
 We use a pinned version of `@mdn/browser-compat-data`, because their [SemVer policy](https://github.com/mdn/browser-compat-data#semantic-versioning-policy) allows for breaking changes to the data structure even in minor and patch releases. If you need to use more up to date data, use the `overrides` facility of `package.json` to specify a later version - but be aware that it might break.
 
+## Options
```

Comment:
The existing option is documented by example/comment in the subpackage's readme, for reasons including: it's specific to the plugin (not the CLI tool), and that's what shows up on the NPM page.

Could these be moved there and follow the same pattern?